packages/create-cloudflare/package.json

```
@@ -84,5 +84,6 @@
 	},
 	"engines": {
 		"node": ">=18.14.1"
-	}
+	},
+	"workers-sdk:prerelease": true
```

Maybe this? Not sure how likely it is that we ever add anything else like this? Code coverage reporting? Opt-in/out of changesets? Release alerts in chat?

```suggestion
	"workers-sdk": {
		"prerelease": true
	}
```